values.yaml

```
@@ -12,6 +12,16 @@ global:
   tlsDisable: true
   # Beta Feature: If deploying to OpenShift
   openshift: false
+  # Create PodSecurityPolicy for pods
+  psp:
+    enable: false
+    # Annotation for PodSecurityPolicy.
+    # This is a multi-line templated string map
```

Maybe mention something like this here?
```suggestion
    # This is a multi-line templated string map, and can also be set as YAML.
```